.gitpod/start_repo.sh

```diff
@@ -26,7 +26,7 @@ if [ -d ${GITPOD_REPO_ROOT}/${reponame} ]; then
   # Now that composer install has been done, if we were using an empty
   # .ddev/config.yaml, we'll do a real ddev config
   if [ ! -s .ddev/config.yaml ]; then
-    ddev config --auto
+    ddev config --auto --project-name="ddev"
```

Comment:
We don't want a project name of ddev, we want the project name based on the github repo, like you had before. And it would be just `ddev config --project-name="<name>"`, don't need the auto if you provide other params.

Reply:
I liked the other way better :)